dags/resources/stages/seed/data/token_amendments.csv

```diff
@@ -7,16 +7,19 @@ address,symbol,name,decimals
 0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9,AAVE,Aave Token,18
 0x1f9840a85d5af5bf1d1762f925bdaddc4201f984,UNI,Uniswap,18
 0x4fabb145d64652a948d72533023f6e7a623c7c53,BUSD,Binance USD,18
-0x0000000000085d4780b73119b644ae5ecd22b376,TrueUSD,TrueUSD,18
+0x0000000000085d4780b73119b644ae5ecd22b376,TUSD,TrueUSD,18
 0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e,YFI,yearn.finance,18
 0xc00e94cb662c3520282e6f5717214004a7f26888,COMP,Compound,18
 0x04fa0d235c4abf4bcf4787af4cf447de572ef828,UMA,UMA Voting Token v1,18
 0xd26114cd6ee289accf82350c8d8487fedb8a0c07,OMG,OMGToken,18
-0x111111111117dc0aa78b770fa6a738034120c302,CHI,Chi Gastoken by 1inch,0
+0x0000000000004946c0e9F43F4Dee607b0eF1fA1c,CHI,Chi Gastoken by 1inch,0
```

Comment:
Could you please lower-case this address? We have a convention in this repo that all addresses are lower-cased for convenience

Reply:
Sorry for this, took it from Etherscan. I'll be more attentive further